README.md

```diff
@@ -114,40 +114,186 @@ start working.
+
+<img src="https://user-images.githubusercontent.com/6227720/115696486-a4ab3580-a35a-11eb-9412-6282963a279d.png" width="600px" alt="Add cluster button. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
 
 Choose a name for your cluster. This name is not related to the Google
 Kubernetes Engine cluster you selected when you deployed the application.
 This name is only used to show your cluster in the Jetstack Secure
-dashboard.
-
-After picking a name, click on the check mark:
+dashboard. Cluster names can only contain alphanumeric values, dots and
+underscores.
 
-<img src="https://user-images.githubusercontent.com/2195781/109155511-ec08b180-776f-11eb-82e1-45b0c32720d6.png" width="600px" alt="Choose your cluster name and click the check mark button. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
+After picking a name, press the "Save cluster name" button:
 
-Next, click the button "Copy command to clipboard":
+<img src="https://user-images.githubusercontent.com/6227720/115697669-a6c1c400-a35b-11eb-851f-ee1e962a8df2.png" width="600px" alt="Choose your cluster name and click the save cluster name button. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
 
-<img src="https://user-images.githubusercontent.com/2195781/109026248-d76ce080-76bf-11eb-94be-bc1c8f54b2cf.png" width="600px" alt="Click the button 'Copy command to clipboard'. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
+At this stage, you have two options depending on which Kubernetes namespace you installed Jetstack Secure in. If you chose the `jetstack-secure` namespace, you can click the "Copy Installation Command To Clipboard" button, which will be enabled once a cluster name is set. This command contains manifests to create a namespace, configuration & secrets to be used by the agent as well as the manifests to install the agent itself.
 
-With the help of a text editor, paste the content and edit it:
+As the installation of the agent has been performed by the google cloud marketplace steps earlier, the only parts of this command you need are the `ConfigMap` and `Secret` resources. Using a text editor, isolate those parts of the command and apply them, excluding the namespace and agent manifests:
 
-1. remove the 6 first lines,
-2. remove the last line.
-
-<img src="https://user-images.githubusercontent.com/2195781/109153775-a21ecc00-776d-11eb-89d0-6beea71c7c07.png" width="600px" alt="Click the button 'Open a text editor and change a few lines from the configmap and secret. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-config
+  namespace: jetstack-secure
```

Comment:
I'm happy to include this here, however the ns is set in the file and in the following command. I guess we want to pick one, probably the the command.

Reply:
Yeah, this is one of the parts where it gets confusing. Since this sample is true to what you get from the UI, it includes the namespace. 

I think it makes sense to remove the namespace from this sample, but should the instructions also state to remove the namespace keys? I already mentioned changing the namespace keys if required in the instructions.

Reply:
I think drop the keys here. Then the ns will only be set in the apply command.